code/modules/clothing/spacesuits/_spacesuits.dm

```diff
@@ -159,10 +159,10 @@
 
 /// Open the cell cover when ALT+Click on the suit
 /obj/item/clothing/suit/space/AltClick(mob/living/user)
-	. = ..()

```

Comment:
this can definitely be removed though since it did literally nothing

Reply:
resolved